ui/components/app/toast-master/toast-master.js

```diff
@@ -0,0 +1,299 @@
+          />
+        }
+        text={t('accountIsntConnectedToastText', [
+          account?.metadata?.name,
+          getURLHost(activeTabOrigin),
+        ])}
+        actionText={t('connectAccount')}
+        onActionClick={() => {
+          // Connect this account
+          dispatch(addPermittedAccount(activeTabOrigin, account.address));
+          // Use setTimeout to prevent React re-render from
+          // hiding the tooltip
+          setTimeout(() => {
+            // Trigger a mouseenter on the header's connection icon
+            // to display the informative connection tooltip
+            document
+              .querySelector(
+                '[data-testid="connection-menu"] [data-tooltipped]',
+              )
+              ?.dispatchEvent(new CustomEvent('mouseenter', {}));
+          }, 250 * MILLISECOND);
+        }}
+        onClose={() => setHideConnectAccountToast(true)}
+      />
+    )
+  );
+}
+
+function SurveyToastMayDelete() {
+  const t = useI18nContext();
+
+  const showSurveyToast = useSelector(selectShowSurveyToast);
+
+  return (
+    showSurveyToast && (
+      <Toast
+        key="survey-toast"
+        startAdornment={
+          <Icon name={IconName.Heart} color={IconColor.errorDefault} />
+        }
+        text={t('surveyTitle')}
+        actionText={t('surveyConversion')}
+        onActionClick={() => {
+          global.platform.openTab({
+            url: SURVEY_LINK,
+          });
+          setSurveyLinkLastClickedOrClosed(Date.now());
+        }}
+        onClose={() => {
+          setSurveyLinkLastClickedOrClosed(Date.now());
+        }}
+      />
+    )
+  );
+}
+
+function PrivacyPolicyToast() {
+  const t = useI18nContext();
+
+  const { showPrivacyPolicyToast, newPrivacyPolicyToastShownDate } =
+    useSelector(selectShowPrivacyPolicyToast);
+
+  // If the privacy policy toast is shown, and there is no date set, set it
+  if (showPrivacyPolicyToast && !newPrivacyPolicyToastShownDate) {
+    setNewPrivacyPolicyToastShownDate(Date.now());
+  }
+
+  return (
+    showPrivacyPolicyToast && (
+      <Toast
+        key="privacy-policy-toast"
+        startAdornment={
+          <Icon name={IconName.Info} color={IconColor.iconDefault} />
+        }
+        text={t('newPrivacyPolicyTitle')}
+        actionText={t('newPrivacyPolicyActionButton')}
+        onActionClick={() => {
+          global.platform.openTab({
+            url: PRIVACY_POLICY_LINK,
+          });
+          setNewPrivacyPolicyToastClickedOrClosed();
+        }}
+        onClose={setNewPrivacyPolicyToastClickedOrClosed}
+      />
+    )
+  );
+}
+
+function SwitchedNetworkToast() {
+  const t = useI18nContext();
+  const dispatch = useDispatch();
+
+  const switchedNetworkDetails = useSelector(getSwitchedNetworkDetails);
+  const switchedNetworkNeverShowMessage = useSelector(
+    selectSwitchedNetworkNeverShowMessage,
+  );
+
+  const show = switchedNetworkDetails && !switchedNetworkNeverShowMessage;
```

Comment:
`show`, being a verb, seems more likely to be referring to a function rather than a boolean